docs/UserGuide.adoc

```diff
@@ -397,6 +397,32 @@ Expected Outcome:
 
 // end::lend[]
 
+// tag::paid[]
+==== Recording the money your friend paid back: `paid`
+
+Records that your friend, who owed you money has paid you back. +
+Format: `people paid <person's index> [<debt's index>]`
+
+****
+* The <person's index> refers to the index number shown before a displayed person.
+* The <debt's index> refers to the index number shown before the debt under the specified person.
+* Both indexes *must be a positive integer* 1, 2, 3, ...
+****
+
+[TIP]
+The debt's index is optional.
+All debts will be removed from the indicated person if the debt's index is not specified.
+
+Examples:
+
+* `people paid 2 1`
```

Comment:
i thought it's "2.1" haha

Reply:
Wait but since debt's index is optional, i think 2 1 better. 

Also, I think you should change 'debt' to 'loan'

Reply:
I used debt because we used debt for both types throughout our UG. Or, should I change everything related to lend to loan?

Reply:
i think should change haha. if not can be quite confusing